### DISCUSSION TITLE
Supported tokens

### DISCUSSION THREAD

**Discussion by Shreyanshmmgn** [mushroomlabs/hub20#128] [General]
I was wondering how many ERC 20 tokens are supported by hub20 USDT, USDC, BNB etc ?

**Comment by lullis** [comment]
Hey! You can configure the instance as many ERC20 tokens as you wish, it is part of the [environment variables](https://docs.hub20.io/reference/configuration/#hub20_tracked_tokens). As long as the token contract provides the basic interface and emits transfer events, it can be supported by the payment gateway.
